src/main/java/build/buildfarm/worker/Utils.java

```diff
@@ -36,4 +40,13 @@ public static FileStatus statIfFound(Path path, boolean followSymlinks, FileStor
       throw new IllegalStateException(e);
     }
   }
+
+  @SuppressWarnings("units")
+  public static @s(Prefix.micro) long stopwatchToMicroseconds(Stopwatch sw) {
+    return sw.elapsed(MICROSECONDS);
+  }
```

Comment:
Convenience method to convert a stopwatch to a `long` annotated as microseconds.